### ISSUE TITLE
Update to Python 3.13 for RTD builds when available

### ISSUE THREAD

**Issue by funkyfuture** [funkyfuture/deck-chores#148]
> So I did some digging and the website itself still uses the `_` for `deck-chores.daily_job.command` but the `usage.rst` does not. 

> Would you be able to push the changes to the RTD to reflect your already committed changes after merging #147?

_Originally posted by @tabedzki in https://github.com/funkyfuture/deck-chores/issues/146#issuecomment-2411811734_

**Comment by funkyfuture** [comment]
this is currently blocked by: https://github.com/readthedocs/readthedocs.org/issues/11671

when unblocked, i shall then switch to the single-version-option and possibly make `main` the default version.

**Comment by funkyfuture** [comment]
you're right. would you be so kind to open a PR that lowers the required Python version to `3.12` for the RTD build and the package metadata with accompanying `# TODO` hints to this issue. that should suffice from the top of my head.

**Comment by funkyfuture** [comment]
okay, eventually it works. please verify that it "works for you".

**Comment by tabedzki** [comment]
The website https://deck-chores.readthedocs.io/ works for me! Thanks for changing the requirements.txt and adding the `# TODO` I missed

**Comment by tabedzki** [comment]
> this is currently blocked by: [readthedocs/readthedocs.org#11671](https://github.com/readthedocs/readthedocs.org/issues/11671)
> 
> when unblocked, i shall then switch to the single-version-option and possibly make `main` the default version.

I just checked: the blocking issue has been closed
